tools/all_questions.xml

```diff
@@ -118,8 +118,8 @@
 		<questionset dc:uri="https://rdmorganiser.github.io/terms/questions/ua_ruhr/general/topic-research_field"/>
 		<is_collection>True</is_collection>
 		<order>0</order>
-		<help lang="en">The list of disciplines follows the &lt;a href="http://www.dfg.de/en/dfg_profile/statutory_bodies/review_boards/subject_areas/index.jsp" target="_blank"&gt;subject classification of the DFG (German Research Foundation)&lt;/a&gt;.</help>
-		<help lang="de">Die Liste der Disziplinen entspricht der &lt;a href="http://www.dfg.de/dfg_profil/gremien/fachkollegien/faecher/" target="_blank"&gt;Fachsystematik der Deutschen Forschungsgemeinschaft (DFG)&lt;/a&gt;.</help>
+		<help lang="en">The list of disciplines follows the &lt;a href="http://www.dfg.de/en/dfg_profile/statutory_bodies/review_boards/subject_areas/index.jsp" target=_blank"&gt;subject classification of the DFG (German Research Foundation)&lt;/a&gt;.</help>
+		<help lang="de">Die Liste der Disziplinen entspricht der &lt;a href="http://www.dfg.de/dfg_profil/gremien/fachkollegien/faecher/" target=_blank"&gt;Fachsystematik der Deutschen Forschungsgemeinschaft (DFG)&lt;/a&gt;.</help>
```

Comment:
The value for `target`  only has one quotation mark now. Is that intended?

Reply:
no it is not. Introduced from upstream. I opened an issue https://github.com/rdmorganiser/rdmo-catalog/issues/15

Reply:
see 5ae5cb6 and 27f82be